twins/users/users.go

```diff
@@ -0,0 +1,45 @@
+//
```

Comment:
What is this? Why users? I mean why Twins servis has a differnet user than Mainflux?